setup.py

```diff
@@ -5,7 +5,7 @@
     long_description = fh.read()
 
 dask_deps = ["dask>=2.22.0", "distributed>=2.22.0"]
-ray_deps = ["ray[default]>=1.4.0", "pyarrow>=1.0"]
+ray_deps = ["ray[default]>=1.4.0", "pyarrow>=6.0"]
```

Comment:
An option for pinning pyarrow would be >= 4.0.1 like Ray requires it as extras ([link])(https://github.com/ray-project/ray/blob/master/python/setup.py#L197). However, our requirements on pyarrow version for modin-ray and modin-omnisci should be aligned. Any thoughts on what a better option would be?